modules/op-signing-secrets-in-tekton-chains.adoc

```diff
@@ -39,6 +39,62 @@ Cosign prompts you for a password, and creates a Kubernetes secret.
 
 . Store the encrypted `cosign.key` private key and the `cosign.password` decryption password in the `signing-secrets` Kubernetes secret. Ensure that the private key is stored as an encrypted PEM file of the `ENCRYPTED COSIGN PRIVATE KEY` type.
 
+
+[id="chains-signing-secrets-skopeo_{context}"]
+== Signing using skopeo
+
+You can generate keys using skopeo and use them in the `cosign` signing scheme with {tekton-chains}:
+
+. Install link:https://github.com/containers/skopeo[skopeo].
+
+. Generate a public/private key pair.
++
+[source,terminal]
+----
+$ skopeo generate-sigstore-key --output-prefix <mykey>
+----
++
+Replace `<mykey>` with a key name of your choice. Skopeo prompts you for a passphrase for the private key. It then creates the key files named `<mykey>.private` and `<mykey>.pub`.
+. Encode the key files using base64.
++
+[source,terminal]
+----
+$ base64 -w 0 <mykey>.pub > b64.pub
+$ base64 -w 0 <mykey>.private > b64.private
+----
++
+. Encode the passhprase using base64.
++
+[source,terminal]
+----
+$ echo -n '<passphrase>' | base64 -w 0 > b64.passphrase
+----
++
+. Create and then edit the `signing-secrets` secret in your {pipelines-title} namespace.
++
+[source,terminal]
+----
+$ oc create secret generic signing-secrets -n openshift-pipelines
+$ oc edit secret -n openshift-pipelines signing-secrets
+----
++
+Add the encoded keys in the data of the secret in the following way:
++
+[source,yaml]
+----
+apiVersion: v1
+data:
+  cosign.key: <Encoded <mykey>.private>
+  cosign.password: <Encoded passphrase>
+  cosign.pub: <Encoded <mykey>.pub>
+immutable: true
+kind: Secret
+metadata:
+  name: signing-secrets
+# ...
+type: Opaque
+----
+
```

Comment:
This is a procedure and needs to have a separate module - it can't be included with a conceptual procedure